src/constants/externalResources.ts

```diff
@@ -38,12 +38,6 @@ export const getJiraBugUrl = (jiraHost: string) =>
 export const getJiraImprovementUrl = (jiraHost: string) =>
   `https://${jiraHost}/secure/CreateIssueDetails!init.jspa?pid=22786&issuetype=4&priority=4&labels=user-feedback`;
 
-export const legacyRoutes = {
-  distros: "/distros",
-  hosts: "/spawn",
-  projects: "/projects",
-};
```

Comment:
Are these all deprecated routes now?

Reply:
I searched for the usage in the codebase and it came up empty! So it's just an unused variable